hop/subscribe.py

```diff
@@ -51,9 +84,9 @@ def _add_parser_args(parser):
     parser.add_argument(
         "-t",
         "--timeout",
-        type=float,
         default=10,
-        help="Specifies the time (in seconds) to wait for new messages. Default: 10s",
+        help="Specifies the time (in seconds) to wait for messages before timing out; "
+        "specify 'None' to wait indefinitely.  Default: 10 seconds",
```

Comment:
I think a more common pattern in unix-y tools would be to specify a negative value to wait indefinitely.

Reply:
Agreed, that was my first thought -- I went with `None` since that was the convention in ADC and ConfluentKafka, though I could change this option on the user's side.